cmd/combine.go

```diff
@@ -23,24 +24,26 @@ func newCombineCmd(runFunc func(ctx context.Context, clusterDir string, force bo
 		Long:  "Combines the private key shares from a threshold of operators in a distributed validator cluster into a set of validator private keys that can be imported into a standard Ethereum validator client.\n\nWarning: running the resulting private keys in a validator alongside the original distributed validator cluster *will* result in slashing.",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runFunc(cmd.Context(), clusterDir, force)
+			return runFunc(cmd.Context(), clusterDir, outputDir, force)
 		},
 	}
 
 	bindCombineFlags(
 		cmd.Flags(),
 		&clusterDir,
+		&outputDir,
 		&force,
 	)
 
 	return cmd
 }
 
-func newCombineFunc(ctx context.Context, clusterDir string, force bool) error {
-	return combine.Combine(ctx, clusterDir, force)
+func newCombineFunc(ctx context.Context, clusterDir, outputDir string, force bool) error {
+	return combine.Combine(ctx, clusterDir, outputDir, force)
 }
 
-func bindCombineFlags(flags *pflag.FlagSet, clusterDir *string, force *bool) {
-	flags.StringVar(clusterDir, "cluster-dir", ".charon/", `Parent directory containing a number of .charon subdirectories from each node in the cluster.`)
+func bindCombineFlags(flags *pflag.FlagSet, clusterDir, outputDir *string, force *bool) {
+	flags.StringVar(clusterDir, "cluster-dir", ".charon/cluster", `Parent directory containing a number of .charon subdirectories from each node in the cluster.`)
+	flags.StringVar(outputDir, "output-dir", "./validator-keys", "Directory to output the combined private keys to.")
```

Comment:
```suggestion
	flags.StringVar(outputDir, "output-dir", "./validator_keys", "Directory to output the combined private keys to.")
```

This might seem astronomically nit picky, but the default staking CLI program, which we are trying to be identical to, uses an _ not a -. If we exactly match, import commands like this will work without edits. 

https://lighthouse-book.sigmaprime.io/validator-import-launchpad.html#1-run-the-lighthouse-account-validator-import-command

Reply:
> astronomically nit picky

Yeah :P

Makes sense though, thanks for pointing this out!